spec/features/background_processor_spec.rb

```diff
@@ -2,78 +2,74 @@
+    remove_project_directory
+    create_dummy_project("background_processor" => true, "heroku" => true)
+  end
 
-    it "adds sidekiq gem to Gemfile" do
-      gemfile_content = IO.read("#{project_path}/Gemfile")
-      expect(gemfile_content).to include("sidekiq")
-    end
+  it "adds sidekiq gem to Gemfile" do
+    gemfile_content = IO.read("#{project_path}/Gemfile")
+    expect(gemfile_content).to include("sidekiq")
+  end
 
-    it "adds queue_adapter to application.rb" do
-      content = IO.read("#{project_path}/config/application.rb")
-      expect(content).to include("config.active_job.queue_adapter = :sidekiq")
-    end
+  it "adds queue_adapter to application.rb" do
+    content = IO.read("#{project_path}/config/application.rb")
+    expect(content).to include("config.active_job.queue_adapter = :sidekiq")
+  end
 
-    it "adds async queue_adapter to development.rb" do
-      content = IO.read("#{project_path}/config/environments/development.rb")
-      expect(content).to include("config.active_job.queue_adapter = :async")
-    end
+  it "adds async queue_adapter to development.rb" do
+    content = IO.read("#{project_path}/config/environments/development.rb")
+    expect(content).to include("config.active_job.queue_adapter = :async")
+  end
 
-    it "adds test queue_adapter to test.rb" do
-      content = IO.read("#{project_path}/config/environments/test.rb")
-      expect(content).to include("config.active_job.queue_adapter = :test")
-    end
+  it "adds test queue_adapter to test.rb" do
+    content = IO.read("#{project_path}/config/environments/test.rb")
+    expect(content).to include("config.active_job.queue_adapter = :test")
+  end
 
-    it "modifies Procfile" do
-      content = IO.read("#{project_path}/Procfile")
-      expect(content).to include("worker: bundle exec sidekiq")
-    end
+  it "modifies Procfile" do
+    content = IO.read("#{project_path}/Procfile")
+    expect(content).to include("worker: bundle exec sidekiq")
+  end
 
-    it "modifies README" do
-      content = IO.read("#{project_path}/README.md")
-      expect(content).to include("this project uses [Sidekiq]")
-    end
+  it "modifies README" do
+    content = IO.read("#{project_path}/README.md")
+    expect(content).to include("this project uses [Sidekiq]")
+  end
 
-    it "adds ENV vars" do
-      content = IO.read("#{project_path}/.env.development")
-      expect(content).to include("DB_POOL=25")
-      expect(content).to include('REDIS_HOST=127.0.0.1')
-      expect(content).to include('REDIS_PORT=COMMAND_EXPAND(make services-port SERVICE=redis PORT=6379)')
-      expect(content).to include('REDIS_URL=redis://${REDIS_HOST}:${REDIS_PORT}/1')
-    end
+  it "adds ENV vars" do
+    content = IO.read("#{project_path}/.env.development")
+    expect(content).to include("DB_POOL=25")
+    expect(content).to include('REDIS_HOST=127.0.0.1')
+    expect(content).to include('REDIS_PORT=COMMAND_EXPAND(make services-port SERVICE=redis PORT=6379)')
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Layout/LineLength: Line is too long. [103/100]